styles.css

```diff
@@ -54,6 +55,11 @@ nav {
 
 .home-icons ul {
   display: flex;
+  list-style: none;
+}
+
+header a {
+  text-decoration: none;
 }
 
 .home-icons li a {
```

Comment:
- [ ] As requested by the previous code reviewer, You are supposed to maintain the features of the mobile version whiles adding the desktop version. However, I cannot see the hamburger menu when you switch to the mobile version. Kindly check and fix it to maintain a professional and up-to-standard design. :pray:
  ![image](https://user-images.githubusercontent.com/92303891/221382237-7deac7f5-1172-4fdf-a7e5-c5af7eb52b3f.png)